CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## unreleased
 * BraintreePayPalNativeCheckout
   * Bump PayPalCheckout to version 1.3.0 with code signing & a privacy manifest file.
+* BraintreePayPalMessaging (BETA)
```

Comment:
It looks like this accidentally got added under the `6.11.0` section at some point but hasn't actually been released. Lets remove it from there and move it under `unreleased` and update the names there:
```
* BraintreePayPalMessaging (BETA)
  * Add `BTPayPalMessagingRequest`, `BTPayPalMessagingColor`, `BTPayPalMessagingLogoType`, `BTPayPalMessagingOfferType`, `BTPayPalMessagingPlacement`, `BTPayPalMessagingTextAlignment`, and `BTPayPalMessagingDelegate`
  * Add `BTPayPalMessagingView` to display PayPal messages to promote offers such as Pay Later and PayPal Credit to customers.
    * To get started create a `BTPayPalMessagingView` and call `start(_:)` with a `BTPayPalMessagingRequest`
```

Reply:
Updated here: f22a9b1d0f9fed0fc33e8e8438ebf082ae5c608b

Reply:
Lets also delete it from the `6.11.0` section since it wasn't actually released there

Reply:
Updated: [ae5a8a4](https://github.com/braintree/braintree_ios/pull/1286/commits/ae5a8a446d3aad3638a3ba7501cbcbdbecacc603)